### PR TITLE
Renamed the asset intelligence to Search Users and Devices

### PR DESCRIPTION
**Pull request by hardikhdholariya**

### CODE REVIEW

File: docs/user_guide/intelligence_dashboard.md [hunk 1]
```diff
@@ -108,4 +113,4 @@ This dashboard displays the following information about devices:
 
 (The screenshot above does not include the full dashboard)
 
-The Asset Intelligence dashboard has been added to the Cyences app in version 1.4.0.
\ No newline at end of file
+The "Search Users and Devices" dashboard has been added to the Cyences app in version 1.4.0.
```
Comment (VatsalJagani) [inline docs/user_guide/intelligence_dashboard.md:116]: The "Search Users and Devices" dashboard has been added to the Cyences app in version 1.4.0, initially named "Asset Intellience", later renamed to "Search Users and Devices" in version 4.3.0.

File: docs/index.md [hunk 2]
```diff
@@ -81,7 +82,7 @@ Cyences has other unique first in the market features on Splunk, like Alert Dige
...
+* **Device Inventory and Search Users and Devices**: Version 1.2.0 of the Cyences app features a new component and that is the Device Inventory Table. It is a vital tool that helps with the security audit process and requires zero configuration. The Device Inventory Table lists all of the different devices present in an environment by correlating data from CrowdStrike, Lansweeper, Kaspersky, Qualys, Sophos, Tenable, Windows Defender, etc. And there is one more dashboard called "Search Users and Devices" which is very helpful for investigating a security incident. For more information, please refer to the [Device Inventory and Search Users and Devices]({{ site.baseurl }}/user_guide/intelligence_dashboard/) section for more details.
```
Comment (VatsalJagani) [inline docs/index.md:85]: You should have made the documentation changes to a separate PR. But its fine, just merge it at the time of release.

File: docs/index.md [hunk 3]
```diff
@@ -81,7 +82,7 @@ Cyences has other unique first in the market features on Splunk, like Alert Dige
...
+* **Device Inventory and Intelligence**: Version 1.2.0 of the Cyences app features a new component and that is the Device Inventory Table. It is a vital tool that helps with the security audit process and requires zero configuration. The Device Inventory Table lists all of the different devices present in an environment by correlating data from CrowdStrike, Lansweeper, Kaspersky, Qualys, Sophos, Tenable, Windows Defender, etc. And there is one more dashboard called "Intelligence" which is very helpful for investigating a security incident. For more information, please refer to the [Device Inventory and Intelligence]({{ site.baseurl }}/user_guide/intelligence_dashboard/) section for more details.
```
Comment (VatsalJagani) [inline docs/index.md:85]: Can we add User Inventory details here as well?

File: docs/user_guide/intelligence_dashboard.md [hunk 4]
```diff
@@ -8,11 +8,16 @@ parent: User Guide
 
 # Intelligence 
 
-Cyences version 1.2.0 introduces a new drop-down option named Intelligence in the app's navigation bar and it contains the following items:
+* Cyences version 1.2.0 introduces a new drop-down option named Intelligence in the app's navigation bar and it contains the following items:
 
-* Asset Intelligence
-* Device Inventory
-* Globally Detected Malicious IPs
+    * Asset Intelligence
```
Comment (VatsalJagani) [inline docs/user_guide/intelligence_dashboard.md:13]: List all new version:
-> device inventory
-> user inventory
-> intelligence
globally detected malicious ips

**Review by VatsalJagani — APPROVED** [review]

File: docs/user_guide/intelligence_dashboard.md (hunk 4 above)
Comment (hardikhdholariya) [reply docs/user_guide/intelligence_dashboard.md:13]: It's version wise high level details. I've added intelligence and user inventory in second point.
image

File: docs/index.md (hunk 3 above)
Comment (hardikhdholariya) [reply docs/index.md:85]: Done

File: docs/user_guide/intelligence_dashboard.md (hunk 4 above)
Comment (hardikhdholariya) [reply docs/user_guide/intelligence_dashboard.md]: Done

**Pull request merged**